aten/src/ATen/native/TensorTransformations.cpp

```diff
@@ -136,7 +136,7 @@ Tensor rot90(const Tensor& self, int64_t k, IntArrayRef dims) {
     case 3:
       return self.flip({dims[0]}).transpose_(dims[0], dims[1]);
     default:
-      return self.clone();
+      return self.clone(at::MemoryFormat::Contiguous);
```

Comment:
Why do we need contig here?

Reply:
It is not obvious that we should use Preserve here, so I decided to go with Contiguous to be safe

Reply:
SGTM